qlib/FixedLengthUtil.qm

```diff
@@ -741,8 +745,13 @@ public class FixedLengthUtil::FixedLengthAbstractWriter {
             if (spacingsize) {
                 string spacing = strmul(fh.padding_char ?? " ", spacingsize);
 
-                if (fh.padding == "left" || (!fh.padding && fh.type == "int"))
```

Comment:
I believe it works the same as before (that is right align for integers is default).
I wrote new tests, to cover this and other cases, that I'm adding to the pull request.